### ISSUE TITLE
Can't add Elementwise Equal Layer in TensorRT 8.0

### ISSUE THREAD

**Issue by 842974287** [NVIDIA/TensorRT#1526]
## Description

I'm trying to add an elementwise layer for equal operation but seeing some errors. Wondering what would be the right way to add such layer into the network.

If I write like this
```
logger = trt.Logger(trt.Logger.VERBOSE)
builder = trt.Builder(logger)
network = builder.create_network()
x = network.add_input("x", shape=(1,), dtype=trt.float32)
y = network.add_input("y", shape=(1,), dtype=trt.float32)
layer = network.add_elementwise(x, y, trt.ElementWiseOperation.EQUAL)
output = layer.get_output(0)
network.mark_output(output)
config = builder.create_builder_config()
engine = builder.build_engine(network, config)
```
I'm seeing the following error.
```
[TensorRT] ERROR: 4: Output tensor (Unnamed Layer* 0) [ElementWise]_output of type Float produced from output of incompatible type Bool
[TensorRT] ERROR: 4: [network.cpp::validate::2604] Error Code 4: Internal Error (Could not compute dimensions for (Unnamed Layer* 0) [ElementWise]_output, because the network is not valid.)
```

I figured maybe I need to set the type of output and I modified the code to
```
logger = trt.Logger(trt.Logger.VERBOSE)
builder = trt.Builder(logger)
network = builder.create_network()
x = network.add_input("x", shape=(1,), dtype=trt.float32)
y = network.add_input("y", shape=(1,), dtype=trt.float32)
layer = network.add_elementwise(x, y, trt.ElementWiseOperation.EQUAL)
output = layer.get_output(0)
output.dtype = trt.bool
network.mark_output(output)
config = builder.create_builder_config()
engine = builder.build_engine(network, config)
```
And I'm getting a different error
```
[TensorRT] ERROR: 2: [elementwiseBuilder.cpp::addSupportedFormats::589] Error Code 2: Internal Error (Assertion kCudaElementWiseSupportsBool failed.Cuda backend does not support kEQUAL/kGREATER/kLESS)
```

Any insight would be really appreciated!

## Environment

**TensorRT Version**: 8.0.1.6
**NVIDIA GPU**: RTX 2070 super
**NVIDIA Driver Version**: 465.19.01
**CUDA Version**: 11.3

**Comment by nvpohanh** [comment]
@842974287 I think this has been fixed in latest TRT (8.4.1). Could you give it a try? If the issue still exists, we will debug this. Thanks

**Comment by nvpohanh** [comment]
Closing due to >14 days without activity. Please feel free to reopen if the issue still exists. Thanks
